src/lightstep-datasource/datasource.js

```diff
@@ -126,31 +136,116 @@ export class LightStepDatasource {
     return this.q.when({});
   }
 
-  metricFindQuery() {
+  metricFindQuery(query) {
+    const interpolated = this.templateSrv.replace(query, null, 'regex');
+
+    let queryMapper = this.defaultMapper();
+    if (interpolated) {
+      queryMapper = this.parseQuery(interpolated);
+    }
+
     return this.doRequest({
-      url: `${this.url}/public/v0.1/${this.organizationName}/projects/${this.projectName}/searches`,
+      url: `${this.url}/public/${version}/${this.organizationName}/projects/${this.projectName}/streams`,
       method: 'GET',
     }).then(response => {
-      const searches = response.data.data;
-      return _.flatMap(searches, search => {
-        const attributes = search["attributes"];
+      const streams = response.data.data;  
+      return _.flatMap(streams, stream => {
+        const attributes = stream["attributes"];
         const name = attributes["name"];
         const query = attributes["query"];
```

Comment:
`stream_query` - now that there are a number of function parameters called `query`, I think this will help make the calls less ambiguous